apps/src/pythonlab/pyodideRunner.ts

```diff
@@ -61,7 +64,20 @@ export async function handleRunClick(
     }
     await runPythonCode(code, source);
     if (isNeighborhoodLevel()) {
-      CodebridgeRegistry.getInstance().getNeighborhood()?.onClose();
+      const neighborhood = CodebridgeRegistry.getInstance().getNeighborhood();
+      neighborhood?.onClose();
+      const projectManager = Lab2Registry.getInstance().getProjectManager();
```

Comment:
Can we extract the svg logic out to a helper method? I.e. here we will just call something like `saveThumbnail`